android/src/main/java/com/thebluealliance/androidclient/datafeed/deserializers/EventDeserializer.java

```diff
@@ -16,6 +16,14 @@
 
 public class EventDeserializer implements JsonDeserializer<Event> {
 
+    /**
+     * Returns true if the given element is null or JsonNull. This is handy for checking the result
+     * of JsonObject#get(), which is null if the requested key is absent.
+     */
+    static boolean isNull(JsonElement element) {
```

Comment:
This helper function  might be good to break out somewhere else and use in the other deserializers, as well